pkg/doc/verifiable/common.go

```diff
@@ -73,7 +73,7 @@ func (tid *TypedID) MarshalJSON() ([]byte, error) {
 
 	alias := (*Alias)(tid)
 
-	data, err := marshalWithExtraFields(alias, tid.ExtraFields)
+	data, err := marshalWithExtraFields(alias, tid.CustomFields)
```

Comment:
We should change this to match `CustomFields`

Reply:
I was thinking about this renaming and came to the conclusion that `marshalWithExtraFields()` reflects better the purpose of the function - marshall a struct with adding extra fields to the JSON.

Reply:
If we're replacing the term "extra fields" with "custom fields" everywhere, I think this should be renamed for consistency reasons. It's also equally valid t say the function Marshalls a struct and adds custom fields to the JSON.

Reply:
Completed the renaming! Now it's more consistent, thanks!